src/redux/selectors.ts

```diff
@@ -166,6 +166,13 @@ export const fileIncludesSelector = createSelector(
   }
 );
 
+export const supportedExtensionsSelector = createSelector(
```

Comment:
Do we need a selector if we are returning directly the `supportedExtensions` from the state? 🤔

Reply:
removed